### PR TITLE
Add docs for t-z components

### PR DESCRIPTION
**Pull request by ethanshar**
## Description
Add docs for t-z components

## Changelog
Add docs for t-z components

### CODE REVIEW

File: src/components/text/text.api.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Text",
+  "category": "basic",
+  "description": "A wrapper for Text component with extra functionality like modifiers support",
+  "extends": ["Text"],
```
Comment (lidord-wix) [inline src/components/text/text.api.json:5]: consider adding `extendsLink` for react-native's `Text`

File: src/components/view/view.api.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "View",
+  "category": "basic",
+  "description": "An enhanced View component",
+  "extends": ["View"],
+  "extendsLink": ["https://reactnative.dev/docs/view"],
+  "modifiers": ["margins", "paddings", "alignments", "background", "borderRadius"],
+  "example": "https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/ViewScreen.js",
+  "images": [],
```
Comment (lidord-wix) [inline src/components/view/view.api.json:9]: can be removed

**Review by lidord-wix — COMMENTED** [review]
Cool, I left two small comments..

File: src/components/view/view.api.json (hunk 2 above)
Comment (ethanshar) [reply src/components/view/view.api.json:9]: I kept images array empty so it will remind us to add images to this component

**Review by lidord-wix — APPROVED** [review]

**Pull request merged**